src/main/avro/com/google/BlockToBq/BQRow.avsc

```diff
@@ -10,7 +10,8 @@
     {"name": "difficulty", "type": "long"},
     {"name": "nonce", "type": "long"},
     {"name": "version", "type": "long"},
-    {"name": "work", "type": "long"}
+    {"name": "work", "type": "long"},
+    {"name": "etl_notes", "type": [ "string", "null" ], "default": "null"}
```

Comment:
should the field be etl_notes or to add the following fields:

work_error
pubkey_error
script_error
....etc